src/main/java/games/strategy/engine/framework/map/download/DownloadUtils.java

```diff
@@ -102,6 +103,7 @@ private DownloadUtils() {}
   private static CloseableHttpClient newHttpClient() {
     return HttpClients.custom()
         .setDefaultRequestConfig(RequestConfig.custom().setCookieSpec(CookieSpecs.IGNORE_COOKIES).build())
+        .setRedirectStrategy(new LaxRedirectStrategy())
```

Comment:
I'm curious why `DefaultRedirectStrategy` (the default redirect strategy used by `HttpClientBuilder`) is not sufficient here?  `LaxRedirectStrategy` seems to only be necessary when the request is a POST or PUT.  This class only ever makes `GET` and `HEAD` requests, which appear to be handled correctly by `DefaultRedirectStrategy`.

BTW, the code in this class was tested to follow redirects of `GET` and `HEAD` requests as part of the testing for #1718.